src/applications/disability-benefits/all-claims/utils/validations.js

```diff
@@ -2,19 +2,21 @@ import { convertToDateField } from '~/platform/forms-system/src/js/validation';
 import { isValidDateRange } from '~/platform/forms/validations';
 
 const messages = {
-  invalidRange: 'Enter a service start date that’s before your end date.',
-  endDate1990: 'Enter a service end date after August 2, 1990.',
-  endDate2001: 'Enter a service end date after September 11, 2001.',
+  invalidRange:
+    'Enter a service start date that occurs earlier than your end date',
+  endDate1990: 'Enter a service end date after August 2, 1990',
+  endDate2001: 'Enter a service end date after September 11, 2001',
   // This is largely redundant because the component validation already prevents
   // inputting dates in the future. However, this is in place just in case, because...
   // TODO: The current schema validation says that the Veteran can input a year
   // up to 100 years in the future. That is incorrect and should be corrected/removed.
-  futureDate: 'Enter a service year that’s before or during the current year.',
+  futureDate: 'Enter a service date no later than today’s date',
```

Comment:
Specify service **end** date to match [comps](https://www.figma.com/design/uffC3A5H3rwEtjSybqG84G/526ez---Source-of-truth-updated-5.2025?node-id=1282-33795&p=f&t=aQlsoVVJG9O2hlTK-0):
```suggestion
  futureDate: 'Enter a service end date no later than today’s date',
```
Also consider renaming this and the newly added `beforeDate` key to better represent respective values.